flair/data.py

```diff
@@ -588,7 +588,12 @@ def __copy__(self):
         return s
 
     def __str__(self) -> str:
-        return 'Sentence: "{}" - {} Tokens'.format(' '.join([t.text for t in self.tokens]), len(self))
+
+        if len(self.labels) == 0:
+            return f'Sentence: "{self.to_tokenized_string()}" - {len(self)} Tokens'
+        else:
+            return f'Sentence: "{self.to_tokenized_string()}" - {len(self)} Tokens - Labels: {self.labels} '
```

Comment:
Maybe factorize the first part of the string? And add ` - Labels: {self.labels}` only `if self.labels` ?

Reply:
thanks! for readability purposes though I'd prefer an explicit if-else!